src/train.py

```diff
@@ -72,11 +77,11 @@
 
 if opt.epoch != 0:
     # Load pretrained models
-    encoder.load_state_dict(torch.load("saved_models/%s/encoder_%d.pth" % (opt.model_name, opt.epoch)))
-    G1.load_state_dict(torch.load("saved_models/%s/G1_%d.pth" % (opt.model_name, opt.epoch)))
-    G2.load_state_dict(torch.load("saved_models/%s/G2_%d.pth" % (opt.model_name, opt.epoch)))
-    D1.load_state_dict(torch.load("saved_models/%s/D1_%d.pth" % (opt.model_name, opt.epoch)))
-    D2.load_state_dict(torch.load("saved_models/%s/D2_%d.pth" % (opt.model_name, opt.epoch)))
+    encoder.load_state_dict(torch.load("saved_models/%s/encoder_%02d.pth" % (opt.model_name, opt.epoch)))
```

Comment:
Any reason why using `%02d`? This will limit the number of epochs to be 99 max.

Reply:
Its just for file name formatting purposes so that in the default version it names files with 00, 01, 02, ...., 99. Its just a small touch so things are easily sorted by name when it comes to loading files from the directory.

If someone were to go over epoch 99, it will still take it as it would previously and will just surpass the formatting constraint. Like 100, 101, ... 100000.

I think it's a neat touch for at least the initial parameters (and think it would be overkill to make digit formatting flexible for any n_epoch) - I could remove the %02d if you prefer.